src/databrowser.js

```diff
@@ -132,13 +131,14 @@ define([
                 state: {opened: !node.recursive && children.length <= MAX_OPEN_NODE},
                 children: children,
                 data: {
-                    handleDrop: _.partial(handleDrop, path, node.sourceInfo),
+                    handleDrop: _.partial(handleDrop, node, node.sourceInfo),
                     getNodes: node.recursive ? getNodes : null,
                 }
             };
         }
-        function handleDrop(path, info, target) {
-            var widget = widgets.util.getWidget(target, vellum);
+        function handleDrop(node, info, target) {
+            var widget = widgets.util.getWidget(target, vellum),
+                path = widget.mug.form.richText && node.hashtag || node.xpath;
```

Comment:
I guess in this case if `node.hashtag` happened to be empty it would fall back to `node.xpath`, which is probably what you want, although that syntax is still probably too tricky to use for that intent.